test/Renci.SshNet.IntegrationTests/Renci.SshNet.IntegrationTests.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net48;net8.0</TargetFrameworks>
+    <TargetFrameworks>net48;net9.0</TargetFrameworks>
```

Do we want this?

---

Let's keep a net8.0 target just for local testing

```suggestion
    <TargetFrameworks>net48;net8.0;net9.0</TargetFrameworks>
```